plugin/plugins/jaeger/jaeger.go

```diff
@@ -0,0 +1,50 @@
+package jaeger
+
+import (
+	"github.com/ipfs/go-ipfs/plugin"
+	opentrace "github.com/opentracing/opentracing-go"
+	config "github.com/uber/jaeger-client-go/config"
+)
+
+// Plugins is exported list of plugins that will be loaded
+var Plugins = []plugin.Plugin{
+	&jaegerPlugin{},
+}
+
+type jaegerPlugin struct{}
+
+var _ plugin.PluginTracer = (*jaegerPlugin)(nil)
+
+func (*jaegerPlugin) Name() string {
+	return "jaeger"
+}
+
+func (*jaegerPlugin) Version() string {
+	return "0.0.1"
+}
+
+func (*jaegerPlugin) Init() error {
+	return nil
+}
+
+//Initalize a Jaeger tracer and set it as the global tracer in opentracing api
+func (*jaegerPlugin) InitGlobalTracer(cfg map[string]interface{}) error {
```

Comment:
we should probably use values from the cfg input in the configuration we do here.

Reply:
Good point. I think instead of using a `cfg` we should define tracer specific options in the plugins themselves, since config values can differ across different tracer implementations.